### PR TITLE
Clarified Language of Email notification options. 

### PR DESCRIPTION
**Pull request by buckybytes**
en_US locale change addresses:

#25390

### CODE REVIEW

**Review by denyskon — CHANGES_REQUESTED** [review]
I think these locale keys need a change also....

**Review by denyskon — APPROVED** [review]

**Review by a1012112796 — APPROVED** [review]

**Comment by denyskon** [comment]
Could you adapt the title of the PR to reflect the changes made

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -957,11 +957,12 @@ confirm_delete_account = Confirm Deletion
 delete_account_title = Delete User Account
 delete_account_desc = Are you sure you want to permanently delete this user account?
 
-email_notifications.enable = Enable Email Notifications
```
Comment (techknowlogick) [inline options/locale/locale_en-US.ini:960]: Is enabled used anywhere else? As disabled is a key, I'm wondering if the matching enabled should be removed.

**Review by techknowlogick — CHANGES_REQUESTED** [review]

File: options/locale/locale_en-US.ini (hunk 1 above)
Comment (buckybytes) [reply options/locale/locale_en-US.ini:960]: I've traced "enabled" and "andyourown" back to the model now, along with some comments and other references to the old values.

**Comment by buckybytes** [comment]
I believe I've found every reference to the old wording in the code.  

[b7cb54c](https://github.com/go-gitea/gitea/pull/29682/commits/b7cb54cd2845d4d16ed44b86cd0c2f776cd4a26a)

Might be a better option if there's more worms in this can.

**Comment by buckybytes** [comment]
As I feared, it all went sideways.  I can't find all the references to the values.  Maybe I'll try to tackle it again when I've had more experience with the project.

**Pull request closed without merging**